lib/bridge/github/issues.rb

```diff
@@ -16,9 +16,9 @@ def issues(label = nil, opts={})
 
       issues_response = gh.issues(params) do |request|
         request.headers["Accept"] = "application/vnd.github.squirrel-girl-preview"
-      end
+      end.all
 
-      issues_response.all.each{
+      issues_response.each{
```

Comment:
:ok: I will bite. What is the difference here? I would expect these to be equivalent.